packages/components/src/components/select-wrapper/select-wrapper-dropdown/select-wrapper-dropdown-utils.ts

```diff
@@ -161,3 +161,19 @@ export const getDropdownVisibility = (
     return isOpen;
   }
 };
+
+export const updateNativePopoverSelectStyles = (
+  host: HTMLElement,
+  optionMaps: OptionMap[],
+  nativePopover: HTMLElement,
+  direction: SelectWrapperDropdownDirection
+): void => {
+  const { left, top, width, height } = host.getBoundingClientRect();
+  const isDirectionDown =
+    direction === 'down' ||
+    (direction === 'auto' &&
+      determineDropdownDirection(host, getAmountOfVisibleOptionsAndOptgroups(optionMaps)) === 'down');
+  nativePopover.style.left = `${left + window.scrollX}px`;
+  nativePopover.style.top = `${top + window.scrollY + (isDirectionDown && height)}px`;
```

Comment:
```suggestion
  nativePopover.style.top = `${top + window.scrollY + (isDirectionDown ? height : 0)}px`;
```
Maybe this would be better